### ISSUE TITLE
Culture based localisation

### ISSUE THREAD

**Issue by arithon** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#296]
Just as suggested/planned on the CK2 mod version, could we use some cultural localisation in events? Like using Lok'Narash instead of the "raise armies" answer to rebels?
https://wowwiki-archive.fandom.com/wiki/Orcish

**Comment by zumbak04** [comment]
> Just as suggested/planned on the CK2 mod version, could we use some cultural localisation in events? Like using Lok'Narash instead of the "raise armies" answer to rebels?

Yes. It's possible, you have to find the event and add this localization option on triggers.
